### DISCUSSION TITLE
Autofill/cache participant name for comments

### DISCUSSION THREAD

**Discussion by mattprecious** [lukevella/rallly#271] [Ideas]
After filling out the poll, having to re-type your name to leave a comment is an unfortunate extra step. Auto-filling the name field after submitting a poll response seems like an improvement here, and further, saving to and reading from a cookie could be useful for return visits to the poll.

If you're receptive to this but it's low on the priority list, I can look into adding it.

**Comment by lukevella** [comment]
Thanks. I had an implementation of this in the past similar to what you described but I wasn't happy with it and ended up removing it. I do have plans to make this a bit easier. 

> If you're receptive to this but it's low on the priority list, I can look into adding it.

I'm reworking large chunks of the codebase at the moment to make way for more features so I'm unlikely to accept any PRs that will cause conflicts with what I'm working on. In time, this will change but for now, I wouldn't recommend spending time on this.
